ateam_kenobi/src/play_helpers/easy_move_to.cpp

```diff
@@ -171,4 +183,72 @@ void EasyMoveTo::drawTrajectoryOverlay(
   }
 }
 
+std::optional<ateam_msgs::msg::RobotMotionCommand> EasyMoveTo::generateEscapeVelocity(
+  const World & world,
+  const Robot & robot,
+  std::vector<ateam_geometry::AnyShape> obstacles)
+{
+  if (planner_options_.use_default_obstacles) {
+    path_planning::AddDefaultObstacles(world, obstacles);
+  }
+  path_planning::AddRobotObstacles(world.our_robots, robot.id, obstacles);
+  path_planning::AddRobotObstacles(world.their_robots, obstacles);
+  const auto robot_footprint = ateam_geometry::makeDisk(
+    robot.pos,
+    kRobotRadius + planner_options_.footprint_inflation);
+  const auto colliding_obstacle = path_planning::GetCollidingObstacle(robot_footprint, obstacles);
+  if (!colliding_obstacle) {
+    return std::nullopt;
+  }
+
+  const double safe_speed = 0.5;
```

Comment:
I assume this won't change but maybe we want this "safe speed" defined in a single as a sort of constant?

Reply:
It's a good idea. I'm not sure where else this would get used at the moment off the top of my head, but worth keeping an eye out.